src/main/app.js

```diff
@@ -313,6 +414,22 @@ fluid.defaults("gpii.app", {
     }
 });
 
+/**
+ * Retrieves the id of the machine on which the PSP is running and sets it in the
+ * model. In case the machine id cannot be obtained, this will not prevent the app
+ * from starting.
+ * @param psp {Component} The `gpii.app.psp` component.
+ */
+gpii.app.fetchMachineId = function (that) {
+    machineInfo.machineId().then(function (machineId) {
```

Comment:
Please harmonise with whatever work has been done by @stegru on acquiring machine id in the core